lib/Conversion/LLHDToLLVM/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_mlir_conversion_library(MLIRLLHDToLLVM 
+    LLHDToLLVM.cpp
+
+    DEPENDS
+    MLIRLLHDOpsIncGen
+    MLIRLLHDConversionPassIncGen
+)
+
+set(LIBS
+  MLIRLLVMIR
+  MLIRStandardToLLVM
+  MLIRVector
+  MLIRTransforms
+  LLVMCore
+  LLVMSupport
+  )
+
+add_dependencies(MLIRLLHDToLLVM ${LIBS})
+target_link_libraries(MLIRLLHDToLLVM
+  PUBLIC
+  ${LIBS})
```

Comment:
link_libraries are always dependencies, so the add_dependencies is redundant.  Merge the target_link_libraries into LINK_LIBS and drop ${LIBS}